code/modules/mob/living/basic/farm_animals/bee/_bee.dm

```diff
@@ -0,0 +1,338 @@
+#define BEE_TRAY_RECENT_VISIT 200 //How long in deciseconds until a tray can be visited by a bee again
+#define BEE_DEFAULT_COLOUR "#e5e500" //the colour we make the stripes of the bee if our reagent has no colour (or we have no reagent)
+#define BEE_POLLINATE_YIELD_CHANCE 33
+#define BEE_POLLINATE_PEST_CHANCE 33
+#define BEE_POLLINATE_POTENCY_CHANCE 50
+#define BEE_FOODGROUPS RAW | MEAT | GORE | BUGS
+
+/mob/living/basic/bee
+	name = "bee"
+	desc = "Buzzy buzzy bee, stingy sti- Ouch!"
+	icon_state = ""
+	icon_living = ""
+	icon = 'icons/mob/simple/bees.dmi'
+	gender = FEMALE
+	speak_emote = list("buzzes")
+
+	melee_damage_lower = 1
+	melee_damage_upper = 1
+	attack_verb_continuous = "stings"
+	attack_verb_simple = "sting"
+	response_help_continuous = "shoos"
+	response_help_simple = "shoo"
+	response_disarm_continuous = "swats away"
+	response_disarm_simple = "swat away"
+	response_harm_continuous = "squashes"
+	response_harm_simple = "squash"
+
+	mob_size = MOB_SIZE_LARGE
+	pixel_x = -16
+	base_pixel_x = -16
+
+	speed = 1
+	maxHealth = 10
+	health = 10
+	melee_damage_lower = 1
+	melee_damage_upper = 1
+	faction = list(FACTION_HOSTILE)
+	pass_flags = PASSTABLE | PASSGRILLE | PASSMOB | PASSMACHINE
+	mob_size = MOB_SIZE_TINY
+	mob_biotypes = MOB_ORGANIC|MOB_BUG
+	density = FALSE
+	gold_core_spawnable = FRIENDLY_SPAWN
+	can_be_held = TRUE
+	held_w_class = WEIGHT_CLASS_TINY
+	environment_smash  = ENVIRONMENT_SMASH_NONE
+	habitable_atmos = list("min_oxy" = 0, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	basic_mob_flags = DEL_ON_DEATH
+	ai_controller = /datum/ai_controller/basic_controller/bee
+	///he he he, beegent
+	var/datum/reagent/beegent = null
+	///the house we live in
+	var/obj/structure/beebox/beehome = null
+	///our icon base
+	var/icon_base = "bee"
+	var/isqueen = FALSE
```

Comment:
sounds self-explanatory but i'd like if you could DMdoc for statistical purposes. would also prefer if this could be re-named `is_queen` to keep in the `snake_case` standard of this codebase.